bitnami/airflow/templates/_helpers.tpl

```diff
@@ -285,3 +285,20 @@ WARNING: Rolling tag detected ({{ .Values.git.repository }}:{{ .Values.git.tag }
 +info https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/
 {{- end }}
 {{- end -}}
+
+{{/*
+Return  the proper Storage Class
+*/}}
+{{- define "airflow.storageClass" -}}
```

Comment:
Where are you using this?

Reply:
There are some Charts that do not have pvc nor statefulset, so they do not use persistent volumes. In that cases the globals are necessary if the charts has dependencies, so the dependencies need to take the value from the global value, and I created the template to maintain the structure of the other charts.

Reply:
Thanks for the explanation... I guess it doesn't apply to `apache`, `memcached, `metrics-sever`, `nginx`, `nginx-ingress-controller`, `spark`, and `tensorflow-resnet`. These charts **without any dependencies** and they don't use the **storageClass** parameter.

On the other hand, I understand you need to add the `storageClass` to the values.yaml and README.md but I don't understand why you need the macro in charts like this one (airflow)